x-pack/plugins/security_solution/common/endpoint/types/actions.ts

```diff
@@ -205,9 +221,10 @@ export interface HostIsolationResponse {
   action: string;
 }
 
-export interface ResponseActionApiResponse {
-  action?: string; // only if command is isolate or release
-  data: ActionDetails;
+export type RunningProcessesRequestBody = TypeOf<typeof NoParametersRequestSchema.body>;
```

Comment:
You could also create and just export this type from `schema/actions` and then import it here. I prefer having all these schema-based types in the same schema file so it's just one file path to import from.